### ISSUE TITLE
[Android] Expose acquireDrmInfo from AndroidDRM framework

### ISSUE THREAD

**Issue by SabatinoMasala** [TheWidlarzGroup/react-native-video#2510]
# Feature Request

If we have access to the result of [acquireDrmInfo](https://support.google.com/widevine/answer/6072714?hl=en) we can determine the Android device security levels in Javascript land.

## Why it is needed

If you have access to the device security levels, you could request different streams based on this info. Eg. an HD stream for L1 and an SD stream for L3.

### Example usage

```javascript
import { acquireDrmInfo, DEVICE_IS_PROVISIONED, DEVICE_IS_NOT_PROVISIONED, DEVICE_IS_PROVISIONED_SD_ONLY } from 'react-native-video'

const drmInfo = await acquireDrmInfo()
const WVDrmInfoRequestStatusKey = drmInfo.WVDrmInfoRequestStatusKey

if (WVDrmInfoRequestStatusKey === DEVICE_IS_NOT_PROVISIONED) {
  console.error('Hardware / integration error')
}

if (WVDrmInfoRequestStatusKey === DEVICE_IS_PROVISIONED) {
  // Fetch L1 (hd) stream
} else {
  // Fetch L3 (sd) stream
}

```

**Comment by freeboub** [comment]
I develop a similar feature. I will try to create a pull request with it

**Comment by freeboub** [comment]
Implemented on 6.0.0-alpha.2
